gui-tk/wiring.tcl

```diff
@@ -407,8 +407,13 @@ proc textOK {} {
     destroy .textInput
     canvas.moveOffsX 0
     canvas.moveOffsY 0
-    if {[lsearch [availableOperations] $textBuffer]>-1} {
-		addOperationKey $textBuffer
+	if {[lsearch [availableOperations] $textBuffer]>-1} {
+		  set subTextBuffer [string range $textBuffer 1 end]
+		  if {[lsearch [availableOperations] $subTextBuffer]>-1} { # ensure that names escaped with \ can be used as ops or pars and vars as the case may be. for ticket 1291
```

Comment:
I like your approach, but I don't think it will work for backslashes appearing in the middle of textBuffer. Bit of a contrived example, but suppose someone wanted to name their parameter m\in (m∈). They might be surprised to get the min operator.

Another approach might be to do search and replace on \, replacing it by \\.